src/server/pps/server/api_server.go

```diff
@@ -1770,6 +1774,11 @@ func (a *apiServer) validatePipelineRequest(request *pps.CreatePipelineRequest)
 		return errors.Errorf("pipeline name is %d characters long, but must have at most 63: %q",
```

Comment:
Didn't we want to change this to 51?

Also for these comparisons that yield an error message, I recommend writing:

```go
if got, want := len(request.Pipeline.Name), 63; got != want {
        return errors.Errorf("pipeline name is %d characters long, but must have at most %d", got, want)
}
```

(Though I don't bother making it read that well in general.  "pipeline name too long; got %d, want %d".)

Reply:
I think that may have been verbal but not written, but I am okay doing that; the next check will neck it down further anyway.

How do you feel about messages of the form `pipeline name %q is %d characters longer than %d max`?